packages/account/src/Containers/Account/page-overlay-wrapper.tsx

```diff
@@ -34,10 +34,13 @@ const PageOverlayWrapper = observer(({ routes, subroutes }: PageOverlayWrapperPr
         subitems: route_group?.subroutes?.length ? route_group.subroutes.map(sub => subroutes.indexOf(sub)) : [],
     }));
 
-    const onClickClose = React.useCallback(
-        () => (is_next_wallet_enabled ? history.push(shared_routes.wallets) : history.push(shared_routes.traders_hub)),
-        [history, is_next_wallet_enabled]
-    );
+    const onClickClose = React.useCallback(() => {
+        if (location.pathname === shared_routes.passkeys) {
+            passkeysTrackActionEvent({ action: 'close' });
+        }
+
+        is_next_wallet_enabled ? history.push(shared_routes.wallets) : history.push(shared_routes.traders_hub);
```

Comment:
Lets not add this redirect. we will have same route for wallets and tradershub. Wallets team will fix it.
cc: @nijil-deriv

Reply:
@amina-deriv , but it is on production now.
I just implemented the tracking event function call before this

Reply:
Wallets team already fixed it.  we dont need any changes from our side.
cc: @nijil-deriv